components/single-article-main.jsx

```diff
@@ -16,6 +16,28 @@ const SingleArticleMain =()=>{
       
     })
    },[])
+   const handlePlusClick =() =>{
+    setArticle((currArticle)=>{
+    return {...currArticle,votes: currArticle.votes +1}
+    })
+    updateArticleByArticleId(article_id,{inc_votes:1}).catch((err) =>{
+        console.log(err,"---- eror")
```

Comment:
Probably best to take out the console logs, your catch block is working great and bringing the number back down. It would be even better to give the user a message to say why the number isn't changing.